lib/DefinePlugin.js

```diff
@@ -491,7 +501,7 @@ class DefinePlugin {
 									runtimeTemplate,
 									logger,
 									!parser.isAsiPosition(/** @type {Range} */ (expr.range)[0]),
-									parser.destructuringAssignmentPropertiesFor(expr)
+									getObjKeys(parser.destructuringAssignmentPropertiesFor(expr))
```

Comment:
I couldn't find a suitable case. Actually, I think changing it to `null` here would be fine, because when `typeof definitions[key] === "object"`, it will go to `applyDefineObject`, so in `applyDefine`, the value will not be an object.

Reply:
ah sorry, I see. using `getObjKeys` is fine here.